docs/UserGuide.md

```diff
@@ -354,6 +355,22 @@ If `author-config.csv` is not given and the repo has not provide author details
 
 <hr>
 
+#### `groups-config.csv`
+
+Optionally, you can provide a `groups-config.csv`(which should be in the same directory as `repo-config.csv` file) to provide details on any custom groupings for files in specified repositories ([example](groups-config.csv)). It should contain the following columns:
+
+Column Name | Explanation
+----------- | -----------
+Repository's Location | Same as `repo-config.csv`.
+Group Name | Name of the group e.g.,`test`.
+Globs * | The list of file path globs to include for specified group. e.g.,`**/test/*;**.java`.
+<sup>* **Multi-value column**: multiple values can be entered in this column using a semicolon `;` as the separator.</sup>
```

Comment:
Is it supposed to be like that?
![image](https://user-images.githubusercontent.com/1673303/63162917-62decd80-c056-11e9-930f-3f05f766fe5f.png)

Reply:
Hmm didn't notice this as IntelliJ formatted this nicely and not over a new row.. Made the changes